agents/maintenance_scheduler/README.md

```diff
@@ -0,0 +1,237 @@
+  - Handles inquiries about pricing and discounts, including competitor offers.
+  - Requests manager approval for discounts when necessary.
+- **Appointment Scheduling:**
+  - Schedules appointments for planting services (or other services).
+  - Checks available time slots and presents them to the customer.
+  - Confirms appointment details and sends a confirmation/calendar invite.
+- **Customer Support and Engagement:**
+  - Sends via sms or email plant care instructions relevant to the customer's purchases and location.
+  - Offers a discount QR code for future in-store purchases to loyal customers.
+- **Tool-Based Interactions:**
+  - The agent interacts with the user using a set of tools.
+  - The agent can use multiple tools in a single interaction.
+  - The agent can use the tools to get information and to modify the user's transaction state.
+- **Evaluation:**
+  - The agent can be evaluated using a set of test cases.
+  - The evaluation is based on the agent's ability to use the tools and to respond to the user's requests.
+
+#### Tools
+
+The agent has access to the following tools:
+
+## Setup and Installations
+
+### Prerequisites
+
+- Python 3.11+
+- Poetry (for dependency management)
+- Google ADK SDK (installed via Poetry)
+- Google Cloud Project (for Vertex AI Gemini integration)
+
+### Installation
+1.  **Prerequisites:**
+
+    For the Agent Engine deployment steps, you will need
+    a Google Cloud Project. Once you have created your project,
+    [install the Google Cloud SDK](https://cloud.google.com/sdk/docs/install).
+    Then run the following command to authenticate with your project:
+    ```bash
+    gcloud auth login
+    ```
+    You also need to enable certain APIs. Run the following command to enable
+    the required APIs:
+    ```bash
+    gcloud services enable aiplatform.googleapis.com
+    ```
+
+1.  Clone the repository:
+
+    ```bash
+    git clone https://github.com/google/adk-samples.git
+    cd adk-samples/agents/customer-service
+    ```
+
+    For the rest of this tutorial **ensure you remain in the `agents/customer-service` directory**.
+
+2.  Install dependencies using Poetry:
+
+- if you have not installed poetry before then run `pip install poetry` first. Then you can create your virtual environment and install all dependencies using:
+
+  ```bash
+  poetry install
+  ```
+
+  To activate the virtual environment run:
+
+  ```bash
+  poetry env activate
+  ```
+
+3.  Set up Google Cloud credentials:
+
+    - Ensure you have a Google Cloud project.
+    - Make sure you have the Vertex AI API enabled in your project.
+    - Set the `GOOGLE_GENAI_USE_VERTEXAI`, `GOOGLE_CLOUD_PROJECT`, and `GOOGLE_CLOUD_LOCATION` environment variables. You can set them in your `.env` file (modify and rename .env_sample file to .env) or directly in your shell. Alternatively you can edit [customer_service/config.py](maintenance_scheduler/config.py)
+
+    ```bash
+    export GOOGLE_CLOUD_PROJECT=YOUR_ROJECT_NAME_HERE
```

Comment:
PROJECT

Reply:
Fixed. Also made several changes to the README to remove references to the original agent.